### ISSUE TITLE
Not working on OBS v30(?)

### ISSUE THREAD

**Issue by MaysonLedur** [exeldro/obs-move-transition#183]
Filters doesn't show up anymore on OBS 30. There is no support for it yet? Didn't find any post about it...

**Comment by IkeEisenhauer** [comment]
I haven't looked yet, but I do know that every major release of OBS I have
had to reinstall some plug-in to make them show up again. So I
would suspect you might have to do that again for 30.0

Ike

**Comment by MaysonLedur** [comment]
I did a full reinstall of everything... Didn't fix :/

**Comment by exeldro** [comment]
Can you provide an OBS log file with the issue?

**Comment by MaysonLedur** [comment]
[2023-11-15 15-11-47.txt](https://github.com/exeldro/obs-move-transition/files/13367931/2023-11-15.15-11-47.txt)

**Comment by exeldro** [comment]
The log file does not show any indication that it tried to load the plugin, so it is not installed in the correct directory. Can you compare the path from your OBS shortcut with the path you're installing the plugin?

**Comment by MaysonLedur** [comment]
In fact, I did install OBS in a different location. I hadn't thought that this would influence it, as the Move installer doesn't ask where to install. Is there any way to choose the location for OBS in the Move installer?

**Comment by exeldro** [comment]
@MaysonLedur uninstall the plugin and reinstall the plugin and it should ask the location to install

**Comment by MaysonLedur** [comment]
I browsed through the files, and the Move installer placed its files in the location that would typically be the default OBS installation folder. I simply moved the 'data' and 'obs-plugins' folders related to Move content, and that made it work.

However, the Move installer didn't ask me where to install; it just went from 'next' to 'finish' in the installation process.

Anyway, the OBS v30 wasn't the problem! Thank you for the support!
